src/test/resources/sampleReactNative022Project/index.android.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * Sample React Native App
+ * https://github.com/facebook/react-native
+ */
+
+import React, {
+  AppRegistry,
+  Component,
+  StyleSheet,
+  Text,
+  View
```

Comment:
In this case, this is a snapshot of the result of react-native init, so we don't want to modify this code